src/Ryujinx/Ui/RendererWidgetBase.cs

```diff
@@ -584,7 +583,7 @@ public void Exit()
             }
         }
 
-        private void NVStutterWorkaround()
+        private void NvStutterWorkaround()
```

Comment:
```suggestion
        private void NvidiaStutterWorkaround()
```

in `Ryujinx.Headless.SDL2` I suggested `NV` but maybe `Nvidia` do more sense